views/play.erb

```diff
@@ -0,0 +1,142 @@
+
+<!-- STYLES -->
```

Comment:
Wonder if this could perhaps be extracted into an external CSS file just to keep everything nicely organised and so there doesn't need to be any duplicated code across the erb files?